roles/base/tasks/user.yml

```diff
@@ -72,4 +72,4 @@
 
 - name: Add personal configuration
   include_tasks: dotfiles.yml
-  when: base_user.dotfiles | default(false) | bool
+  when: base_user.dotfiles_url is defined
```

Comment:
This already references the `dotfiles_url` option within the base_users list as mentioned in the first comment. Should work after the other change

Reply:
this also works when tested: user without dotfile config after user with config:
```
TASK [base : Add personal configuration] **********************************************************************************************************************************************************************************************************************************************************
skipping: [ot-rpi7]
```